cli/builder/migrate.go

```
@@ -0,0 +1,823 @@
+	if err != nil {
+		fmt.Fprintf(stderr, "warning: open db for worktree repair: %v\n", err)
+		return
+	}
+	defer func() { _ = store.Close() }()
+
+	type pair struct {
+		worktreePath string
+		repoPath     string
+	}
+	var pairs []pair
+	rows, err := store.DB().QueryContext(ctx,
```

golangci-lint (2.12.2)

[error] rows.Err must be checked

(rowserrcheck)